packages/react-devtools-inline/README.md

```diff
@@ -64,6 +64,23 @@ const DevTools = initialize(contentWindow);
 
 ## Examples
 
+### Supporting named hooks
+
+DevTools can display hook "names" for an inspected component, although determining the "names" requires loading the source (and source-maps), parsing the code, and infering the names based on which variables hook values get assigned to. Because the code for this is non-trivial, it's lazy-loaded only if the feature is enabled.
+
+To configure this package to support this functionality, you'll need to provide a prop that dynamically imports the extra functionality:
```

Comment:
Is the dynamic webpack public path override also necessary to get this working in OSS or is that something that's only necessary in `react-devtools-shell`?

Reply:
TBH I'm not sure. Maybe it depends on which version of Webpack other projects are using?

Reply:
If #22267 doesn't land, should we add that as a note in the README as well? It feels a bit unintuitive for developers otherwise (it definitely was for us 😅)

Reply:
Yeah...let's see what happens 😅 I'm not a Webpack expert so maybe there's a better way to do this than what I did.

TBH I don't think anyone (other than us, Code Sandbox, and Replay.io) uses the inline package.